src/main/start-main-application/lens-window/application-window/lens-window-injection-token.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+import type { ClusterFrameInfo } from "../../../../common/cluster-frames";
+
+export interface SendToViewArgs {
+  channel: string;
+  frameInfo?: ClusterFrameInfo;
+  data?: any[];
```

Comment:
nit: this should be `unknown[]` as we should be trying to move away from using `any` as much as possible.

Reply:
Done